Fix customImporterTabViewModel is duplicated when export of Preferences (#10241)
JabRef/jabref: CHANGELOG.md

- We fixed a bug that prevented external group metadata changes from being merged. [#8873](https://github.com/JabRef/jabref/issues/8873)
- We fixed the shared database opening dialog to remember autosave folder and tick. [#7516](https://github.com/JabRef/jabref/issues/7516)
- We fixed an issue where name formatter could not be saved. [#9120](https://github.com/JabRef/jabref/issues/9120)

> - We fixed a bug where after export of Preferences, customImporterTabViewModel is duplicated [#10176](https://github.com/JabRef/jabref/issues/10176)

Reviewer: ```suggestion
- We fixed a bug where after export of Preferences, custom exports were duplicated [#10176](https://github.com/JabRef/jabref/issues/10176)
```
